### DISCUSSION TITLE
profiles.cpp question

### DISCUSSION THREAD

**Discussion by kestreltom** [rnd-ash/ultimate-nag52-fw#36] [Q&A] [answered]
Does anybody know what the following line refers to?

.target_d_rpm = 60,

It is found under the command  " return ShiftCharacteristics { "   at various locations.

Thanks in advance!
TF

**Comment by rnd-ash** [comment, marked as answer]
Ah!

This seems to be left over from my very early profile code. It does nothing now. Ill remove it or re-purpose it in future commits.

**Comment by kestreltom** [comment]
Thanks!

OK then, no prob.

I am trying my best to narrow down the issue of the harsh 3-2 downshift.
If we could write a sub-routine just to handle that issue, I would be happy
to test and refine it.  Aside from that, the last fw iteration is nearly
perfect now imao, excluding the C mode

Thanks,
tf
